zerver/lib/test_classes.py

```diff
@@ -273,6 +273,12 @@ def users_subscribed_to_stream(self, stream_name, realm):
 
         return [subscription.user_profile for subscription in subscriptions]
 
+    def assertUrlServesContentsOfFile(self, url, result):
```

Comment:
hmm, I just follow @showell suggestion in https://github.com/zulip/zulip/issues/1276#issuecomment-232557317

and I think if we name it using underscore notation, the name will be very long